docker/archive/transport.go

```diff
@@ -40,6 +40,7 @@ func (t archiveTransport) ValidatePolicyConfigurationScope(scope string) error {
 }
 
 // archiveReference is an ImageReference for Docker images.
+// archiveReference.destinationRef is optional and can be nil for destinations as well.
```

Comment:
Non-blocking nit: it would be nice to combine it with the field-specific comment below.

Reply:
fixed.